jetpack.php

```diff
@@ -139,7 +139,9 @@ function vip_jetpack_load() {
 		add_filter( 'instagram_cache_oembed_api_response_body', '__return_true' );
 	}
 
-	require_once __DIR__ . '/vip-jetpack/vip-jetpack.php';
+	if ( defined( 'VIP_JETPACK_LOADED_VERSION' ) ) {
```

Comment:
Should we also check that `VIP_JETPACK_LOADED_VERSION` is not a `none` value? https://github.com/Automattic/vip-go-mu-plugins/blob/b12bf1908960fdf9dcd0e52ccb39edb2d2677e99/jetpack.php#L97-L100

Reply:
@rebeccahum good catch, added that.